style.css

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+  display: flex;
+}
+
+nav {
+  width: 98%;
+  height: 60px;
+  background: #3c3a39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.navbar-logo {
+  display: flex;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 700;
+  font-family: Roboto, sans-serif;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navbar-menu {
+  padding-right: 50px;
+}

```

Comment:
- [ ] Continuing to solve the left white space, remove `padding-right: 50px;` from here too.